lib/vue-lib/src/design-system/layout/Stack.vue

```diff
@@ -34,8 +34,11 @@ const Stack = (
   const wrappedChildren = [] as VNode[];
   const children = getSlotChildren(context.slots.default);
   for (let i = 0; i < children.length; i++) {
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    wrappedChildren.push(children[i]!);
+    if (!children[i]) continue;
+
+    // NOTE - ran into a weird errors that only appeared on the built version of the app
+    // but resolved it by adding this Fragment wrapper around each child
+    wrappedChildren.push(h(Fragment, {}, [children[i]]));
```

Comment:
this Fragment wrapper is the magic that fixed it...  Got the idea from https://github.com/vuejs/core/issues/8298 but never was fully able to wrap my head around what exactly is triggering the issue.